ui/src/pages/group-review.js

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright 2020 Verizon Media
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import Header from '../components/header/Header';
+import UserDomains from '../components/domain/UserDomains';
+import API from '../api';
+import styled from '@emotion/styled';
+import Head from 'next/head';
+// there is an issue with next-link and next-css if the css is not present then it doesnt load so adding this
+import 'flatpickr/dist/themes/light.css';
+import CollectionDetails from '../components/header/CollectionDetails';
+import ReviewList from '../components/review/ReviewList';
+import RequestUtils from '../components/utils/RequestUtils';
+import GroupTabs from '../components/header/GroupTabs';
+import NameHeader from '../components/header/NameHeader';
+import Error from './_error';
+import { MODAL_TIME_OUT } from '../components/constants/constants';
+import createCache from '@emotion/cache';
+import { CacheProvider } from '@emotion/react';
+
+const AppContainerDiv = styled.div`
+    align-items: stretch;
+    flex-flow: row nowrap;
+    height: 100%;
+    display: flex;
+    justify-content: flex-start;
+`;
+
+const MainContentDiv = styled.div`
+    flex: 1 1 calc(100vh - 60px);
+    overflow: hidden;
+    font: 300 14px HelveticaNeue-Reg, Helvetica, Arial, sans-serif;
+`;
+
+const GroupsContainerDiv = styled.div`
+    align-items: stretch;
+    flex: 1 1;
+    height: calc(100vh - 60px);
+    overflow: auto;
+    display: flex;
+    flex-direction: column;
+`;
+
+const GroupsContentDiv = styled.div``;
+
+const PageHeaderDiv = styled.div`
+    background: linear-gradient(to top, #f2f2f2, #fff);
+    padding: 20px 30px 0;
+`;
+
+export default class GroupReviewPage extends React.Component {
+    static async getInitialProps(props) {
+        let api = API(props.req);
+        let reload = false;
+        let notFound = false;
+        let error = undefined;
+        const groups = await Promise.all([
```

Comment:
since this is a new code, let's move all of this to the back end and let the page make just one call to the back end to fetch all the group related data in a single call